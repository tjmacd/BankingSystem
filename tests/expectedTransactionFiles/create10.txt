10                      00000 00000000 A 
05 ABCDEFGHIJKLMNOPQRST 00005 00000.01   
00                      00000 00000000   