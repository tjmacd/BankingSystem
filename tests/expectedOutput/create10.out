Please select session type (standard or admin):
Logged in as admin
Please enter account holder's name:
Enter the initial balance:
Account successfully created for ABCDEFGHIJKLMNOPQRST with account number 00005
Logged out